assets/javascripts/discourse/components/chat-vh.js

```diff
@@ -0,0 +1,46 @@
+import { bind } from "discourse-common/utils/decorators";
+import Component from "@ember/component";
+import { throttle } from "@ember/runloop";
+
+const CSS_VAR = "--chat-vh";
+
+export default class ChatVh extends Component {
+  tagName = "";
+
+  didInsertElement() {
+    this._super(...arguments);
+
+    this.setVH();
+
+    (window?.visualViewport || window).addEventListener(
+      "resize",
+      this.setVHThrottler,
+      false
+    );
+  }
+
+  willDestroyElement() {
+    this._super(...arguments);
+
+    (window?.visualViewport || window).removeEventListener(
+      "resize",
+      this.setVHThrottler
+    );
+  }
+
+  @bind
+  setVH() {
+    if (this.isDestroying || this.isDestroyed) {
+      return;
+    }
+
+    const composerVH =
```

Comment:
I'm curious: why the `* 0.01`?

Reply:
We want to use it as the unit for 1% after